src/components/common/ScreenShotRow.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { Box } from 'grommet';
+import ScreenshotCard from '../ScreenshotCard.js';
+
+const ScreenShotRow = ({
+  margin,
+  src0,
+  src1,
+  src2,
```

Comment:
`src` should be an array, it will will scale better and remove some props. same for `label`.